test/wveOLAS.js

```diff
@@ -287,14 +287,19 @@ describe("Wrapped Voting Escrow OLAS", function () {
             // Create and increase locks for both addresses signers[0] and signers[1]
             await ve.createLock(twoOLABalance, lockDuration);
             await ve.increaseAmount(oneOLABalance);
+            let blockNumber = await ethers.provider.getBlockNumber("latest");
             await ve.connect(owner).createLock(twoOLABalance, lockDuration);
             await ve.connect(owner).increaseAmount(oneOLABalance);
             await ve.connect(owner).increaseAmount(oneOLABalance);
-            let blockNumber = await ethers.provider.getBlockNumber("latest");
 
-            // Get past votes of the owner
-            const votesOwner = ethers.BigNumber.from(await wve.getPastVotes(owner.address, blockNumber));
-            expect(Number(votesOwner)).to.greaterThan(0);
+            // The past votes before the lock must be zero
+            let votesOwner = ethers.BigNumber.from(await wve.getPastVotes(owner.address, blockNumber));
+            expect(votesOwner).to.equal(0);
```

Comment:
Show that the wrapped function is correctly returning zero voting power when asked for the block number before the first lock.